hack/e2e-suite/guestbook.sh

```diff
@@ -37,8 +37,9 @@ prepare-e2e
 
 GUESTBOOK="${KUBE_ROOT}/examples/guestbook"
```

Comment:
Yup. It looks for file extensions that could be schemas and attempts to load them as resources (skipping if invalid schema).